### PR TITLE
Move to binaries hosted at wisc.edu

### LINKED ISSUE #1: Binary installation?
Are you intending this to replace https://github.com/chkwon/PATHSolver.jl ?  If so, is there any way to copy the download and installation approach of the path binaries from it (https://github.com/chkwon/PATHSolver.jl/blob/master/deps/build.jl)?  The usability of the package would increase substantially!

### PR DESCRIPTION
**Pull request by odow**
Closes #1

### CODE REVIEW

**Comment by odow** [comment]
Okay, we're passing, but I get segfaults if I try to use LUSOL. @jlperla, it'd be good if you could try out this branch and see what happens. I'm not 100% confident I've wrapped things correctly.

**Comment by jlperla** [comment]
Thanks so much for pushing forward on these features. I will try to test soon

File: src/C_API.jl [hunk 1]
```diff
@@ -506,13 +501,15 @@ function solve_mcp(
...
     status = c_api_Path_Solve(m, info)
-
     X = c_api_MCP_GetX(m)
-
+    # TODO(odow): I don't know why, but manually calling MCP_Destroy was
+    # necessary to avoid a segfault on Julia 1.0 when using LUSOL. I guess it's
+    # something to do with the timing of when things need to get freed on the
+    # PATH side? i.e., MCP_Destroy before other things?
+    c_api_MCP_Destroy(m)
+    m.ptr = C_NULL
```
Comment (odow) [inline src/C_API.jl:498]: I got the segfaults to disappear by doing this. But I don't really know why...

**Comment by odow** [comment]
Merging this for now because it is a strict improvement on what we had before (people can now actually install PATH). Revisiting the LUSOL segfault can happen in a new PR.

**Comment by jlperla** [comment]
@odow   I tested things out on master (after this merge) on both Windows and WSL/Ubuntu and had no issues whatnoever.  Both to lusol (or at least those setup instructions on the README) and the MOI.Silent also worked without setup issues.

**Pull request merged**